app/code/Magento/Email/Model/Transport.php

```diff
@@ -100,7 +110,8 @@ public function sendMessage()
 
             $this->laminasTransport->send($laminasMessage);
         } catch (\Exception $e) {
-            throw new MailException(new Phrase($e->getMessage()), $e);
+            $this->logger->error($e->getMessage());
```

Comment:
I believe it's better to log whole exception, including the stack trace, otherwise it would be too hard to detect where the exception happened
```suggestion
            $this->logger->error($e);
```

Reply:
Changed